src/sentry/static/sentry/app/actionCreators/events.tsx

```diff
@@ -1,24 +1,40 @@
 import {canIncludePreviousPeriod} from 'app/views/events/utils/canIncludePreviousPeriod';
 import {getPeriod} from 'app/utils/getPeriod';
+import {EventsStats, Organization} from 'app/types';
 
-const BASE_URL = org => `/organizations/${org.slug}/events-stats/`;
+const BASE_URL = (org: Organization) => `/organizations/${org.slug}/events-stats/`;
+
+type Options = {
+  organization: Organization;
+  project?: number[];
+  environment?: string[];
+  period?: string;
+  start?: Date;
+  end?: Date;
+  interval?: string;
```

Comment:
We have default props for `interval`, `limit`, `query`... so they'll never need be optional?

Reply:
They can be though, you can call `doEventsRequest` with only an `organization` and it will work.

Reply:
Ok